inngest-core/src/main/kotlin/com/inngest/Comm.kt

```diff
@@ -27,6 +27,10 @@ data class RegistrationRequestPayload(
     val v: String,
 )
 
+enum class InngestSyncResult {
```

Comment:
ignore this for now.
was initially planning to redo the registration but didn't touch it this time.